classes/Root.php

```diff
@@ -1,40 +1,46 @@
-<?php 
-class Root{
+<?php
+
+class Root
+{
     private $controller;
     private $action;
     private $request;
-    public function __construct($request){
+
+    public function __construct(array $request)
+    {
         $this->request = $request;
-        if($this->request['controller'] == ""){
-            $this->controller='home';
-        }else{
+        if ($this->request['controller'] == "") {
```

Comment:
powinno wystarczyć tak:
$this->controller = $this->request['controller'] ?: 'home';